src/config.ts

```diff
@@ -106,10 +107,7 @@ async function loadTailwindConfig(
   let tailwindConfig: RequiredConfig = { content: [] }
 
   try {
-    let pkgFile = localRequire.resolve('tailwindcss/package.json', {
-      paths: [baseDir],
-    })
-
+    let pkgFile = resolveIn('tailwindcss/package.json', [baseDir])
```

Comment:
If I understand correctly, this one does not use the cache but always uses a fresh lookup, right?

Reply:
This is correct. I could cache these but have to conditionalize this on the set of base directories. I _could_ cache this one but, at least on Tailwind UI, it didn't show up in profiles at all.